index.html

```diff
@@ -66,26 +73,29 @@ <h2>I'm Jay</h2>
     </div>
   </section>
   <section class="works_container">
-    <div class="works_title">
+    <div id="works_title">
       <h2 class="name_title">My Recent Works</h2>
       <span class="line"></span>
     </div>
     <div class="cards">
       <div class="card1">
         <img class="gymimg" src="./Images/Gym.svg" alt="Gym image" />
-        <h3 class="card1_title">Multi-Post Stories</h3>
-        <p class="card1_paragraph">A daily section of privately personalized reads; no accounts or signups required.
-          This has been the industry's standard dummy text ever since the 1500s, when an
-          unknown printer took a standard dummy text.
-        </p>
-        <ul class="buttons1">
-          <li><button class="li_buttons1" type="button">css</button></li>
-          <li><button class="li_buttons1" type="button">html</button></li>
-          <li><button class="li_buttons1" type="button">bootstrap</button></li>
-          <li><button class="li_buttons1" type="button">ruby</button></li>
-        </ul>
-        <button class="card1button" type="button">See Project</button>
+        <div class="details_gyming">
+          <h3 class="card1_title">Multi-Post Stories</h3>
+          <p class="card1_paragraph">A daily section of privately personalized reads; no accounts or signups required.
+            This has been the industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons1">
+            <li><button class="li_buttons1" type="button">css</button></li>
+            <li><button class="li_buttons1" type="button">html</button></li>
+            <li><button class="li_buttons1" type="button">bootstrap</button></li>
+            <li><button class="li_buttons1" type="button">ruby</button></li>
+          </ul>
+          <button class="card1button" type="button">See Project</button>
+        </div>
```

Comment:
- [ ] KIndly align your project section to the left to be on the same line as the section title ("My Recent work") so that it matches the Figma design chosen as much as possible👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/186534185-342ca491-f34b-490f-b1e6-f4163e1adc47.png)  | ![image](https://user-images.githubusercontent.com/63473584/186534325-168b0558-7a52-462e-a246-8d69607d9956.png)  |